plan/plan_test.go

```diff
@@ -69,6 +79,14 @@ func TestCalculate(t *testing.T) {
 		{[]Policy{&SyncPolicy{}}, labeledV1, noLabels, empty, labeledV1, labeledV2, empty},
 		// RecordType should be inherited
 		{[]Policy{&SyncPolicy{}}, typedV1, noType, empty, typedV1, typedV2, empty},
+		// If desired TTL is not configured, do not update
+		{[]Policy{&SyncPolicy{}}, ttlV1, ttlV2, empty, empty, empty, empty},
```

Comment:
I think `ttlV2` should contain a correct TTL (defaulted to 300 or read from the current records in the provider). And then this should actually trigger an update from the old TTL to the new TTL if they are different.

Reply:
This is desired behaviour, ignore it.

If an `Endpoint` fetched from a provider has a TTL (which is always the case) but the desired `Endpoint` gathered from a `Service` doesn't specify a TTL then we keep the value from the first record.

To clarify:
* we take the provider's default whenever we don't specify TTLs
* we keep whatever value we had in case we drop a TTL definition from a servive later
* we overwrite any TTL if we define a TTL in an annotation